src/__tests__/Search/__snapshots__/SearchResultsList.test.tsx.snap

```diff
@@ -380,12 +370,9 @@ exports[`SearchResultsList should match snapshot 1`] = `
                     </p>
                   </div>
                 </li>
-                <span
-                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
-                />
```

Comment:
this was altered due to a change in the await/act/run timers implementation. I believe this is an animation that is no longer visible